Services/Services-Testing-Markdown.Rmd

```diff
@@ -14,25 +14,26 @@ knitr::opts_chunk$set(echo = TRUE)
 options(kableExtra.auto_format = FALSE)
 
 # Line below for testing only
-Loc <- "Marr"
+Loc <- "Teviot and Liddesdale"
 
 locality <- Loc
 LOCALITY <- Loc
 lp_path <- "/conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/"
 
+# stop(getwd())
 
-source("Master RMarkdown Document & Render Code/Global Script.R")
-source("Services/2. Services data manipulation & table.R")
-source("Services/3. Service HSCP map.R")
+source("../Master RMarkdown Document & Render Code/Global Script.R")
+source("../Services/2. Services data manipulation & table.R")
+source("../Services/3. Service HSCP map.R")
```

Comment:
See my other comment.

We want to leave it like this, so we can knit (with the option 'use project dir'). However we also have to specify this in the `rmarkdown::render`

```suggestion
source("Master RMarkdown Document & Render Code/Global Script.R")
source("Services/2. Services data manipulation & table.R")
source("Services/3. Service HSCP map.R")
```